lib/pbench/test/unit/server/conftest.py

```diff
@@ -825,6 +826,27 @@ def get_token_func(pbench_admin_token, server_config, rsa_keys):
     )
 
 
+@pytest.fixture()
+def pbench_drb_api_key(client, server_config, create_drb_user):
+    """Valid api_key for the 'drb' user"""
+    key, created_timestamp = generate_api_key(
+        username="drb",
+        user=create_drb_user,
+    )
+    return key, created_timestamp
+
+
+@pytest.fixture()
+def pbench_drb_api_key_invalid(client, server_config, create_drb_user):
+    """Invalid api_key for the 'drb' user"""
+    key, created_timestamp = generate_api_key(
+        username="drb",
+        user=create_drb_user,
+        valid=False,
+    )
+    return key
```

Comment:
It strikes me as odd that the second of these two functions, which seems to mirror the first, doesn't return two values like the first does.  (Perhaps the first shouldn't return its second value?)